content/preferences.xul

```diff
@@ -42,6 +42,16 @@
              value="&notero.preferences.notionDatabaseID;"
              control="notero-notionDatabaseID" />
       <textbox id="notero-notionDatabaseID" preference="pref-notionDatabaseID" />
+      <separator />
+      <label id="notero-zoteroAPIKey-label"
+             value="&notero.preferences.zoteroAPIKey;"
+             control="notero-zoteroAPIKey" />
+      <textbox id="notero-zoteroAPIKey" preference="pref-zoteroAPIKey" />
```

Comment:
I see why the preferences window isn't registering these values. In addition to what you have here, you also need to add a corresponding `<preference>` element within the `<preferences>` element on line 14.

```xml
<preference id="pref-zoteroAPIKey"
            name="extensions.notero.zoteroAPIKey"
            type="string" />
```